index.js

```diff
@@ -60,16 +60,16 @@ export default class CachedImage extends Component {
     return (
       <View>
         {this.props.isBackground ? (
-          <ImageBackground
+          <ImageBackground style={this.props.cachedImageStyles}
```

Comment:
I think the better solution here is to remove the parent View and pass down the styles without using a specific prop. Take a look at the comment on your original issue